image/image.go

```diff
@@ -18,55 +18,52 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"log"
 	"os"
 	"path/filepath"
 
 	"github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/opencontainers/image-tools/logger"
 	"github.com/pkg/errors"
 )
 
 // ValidateLayout walks through the given file tree and validates the manifest
 // pointed to by the given refs or returns an error if the validation failed.
-func ValidateLayout(src string, refs []string, out *log.Logger) error {
-	return validate(newPathWalker(src), refs, out)
+func ValidateLayout(ctx logger.Context, src string, refs []string) error {
```

Comment:
This doesn't make sense. This should just use a regular context. Don't define a new `Context` type.

Please follow the pattern in `containerd`. It is well-healed and does exactly what is needed here.